pkgs/by-name/sh/shadow-pc/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, fetchurl, appimageTools }:
+
+let
+  pname = "shadow-launcher";
+  version = "9.9.10132";
+  src = fetchurl {
+    url = "https://update.shadow.tech/launcher/prod/linux/ubuntu_18.04/ShadowPC.AppImage";
```

Comment:
Do they offer versioned URLs? This looks like a URL that will change over time. When that happens the hash will break and it will be unusable. If they update multiple times a year, then it is likely to be broken most the time in the stable `nixpkgs` release

Reply:
Unfortunately no, at least not to my knowledge. They provide only prod and beta channels, which always contain the latest version of the app. However, I’ll check their Discord to confirm.

Reply:
Thanks, it would be unfortunate if this derivation were to be permanently broken in `nixpkgs` releases

Reply:
Changed to use url with defined version